run.sh

```diff
@@ -15,7 +15,8 @@ fi
 
 exec 2>&1
 
-java -jar /opt/exhibitor/exhibitor.jar \
+java -javaagent:/agents/appdynamics-jvm/javaagent.jar -Dappdynamics.agent.uniqueHostId=$(cat /agents/appdynamics-jvm/uniqueHostId) \
```

Comment:
Until now there was no dependency on Tupage here, or do I understand it wrongly?

If so, I would prefer, that the script checks, if AppDynamics agent exists and if not, just work as before.